introduction.md

```diff
@@ -1 +1 @@
-I've found JavaScript the most useful path so far.
+*Git and *JavaScript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?